README.md

```diff
@@ -1,31 +1,25 @@
-# Portfolio
+# My portfolio
 
-Your starting portfolio, to iterate on.
+# about me
 
-## Learning Objectives
+i have included a brief description about myself and my background and hobbies and interests
 
-- Customise the starting portfolio with your professional details
-- Iterate on your portfolio every module
-- Review your colleague's portfolio
+## projects
 
-## Requirements
+i have included three projects that are deployed on Netlify and Github
```

Comment:
Great job on actually updating your Readme. We will develop this over the next few months so get used to changing this! :)